packages/@aws-cdk/aws-elasticsearch/README.md

```diff
@@ -9,4 +9,75 @@
 ---
 <!--END STABILITY BANNER-->
 
+To create an Elasticsearch domain:
+
+```ts
+import * as es from '@aws-cdk/aws-elasticsearch';
+
+const cluster = new es.Cluster(this, 'Domain', {
```

Comment:
Yes. I will replace most of the references. I still thinking of keeping cluster config though since it configures the master and data nodes. What do you think?

Reply:
Yes `clusterConfig` seems appropriate 👍